File: desktop/src/com/lyeeedar/desktop/DesktopLauncher.java
package com.lyeeedar.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.lyeeedar.GlobalData;
import com.lyeeedar.MainGame;

public class DesktopLauncher {
	public static void main (String[] arg) {
		GlobalData.Global.release = true;
		GlobalData.Global.game = new MainGame();
		GlobalData.Global.applicationChanger = new LwjglApplicationChanger();
		GlobalData.Global.applicationChanger.createApplication();
	}
}

File: desktop/src/com/lyeeedar/desktop/DesktopLauncher.kt
package com.lyeeedar.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.lyeeedar.Global
import com.lyeeedar.MainGame

object DesktopLauncher
{
	@JvmStatic fun main(arg: Array<String>)
	{
		//Global.release = true
		Global.game = MainGame()
		Global.applicationChanger = LwjglApplicationChanger()
		Global.applicationChanger.createApplication()
	}
}
